### PR TITLE
hlf-connector

### PR DESCRIPTION
**Pull request by anandbanik**
Signed-off-by: Anand Banik <anand.banik@walmart.com>

Integrate with Hyperledger Fabric using REST and Kafka and, Block and Chaincode Event emitter.

### CODE REVIEW

**Comment by anandbanik** [comment]
Hi @arsulegai,
Would appreciate if you could confirm your sponsorship. Thanks!
Anand

File: labs/hlf-connector.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Scope of Lab
+This component is design to integrate Hyperledger Fabric with enterprise systems using REST and Kafka messaging (Including Azure EventHub) with support for PDC and Transient.
+
+This component will involve both tranactional (like data ingestion) and administration (like channel update) related features.
+
+# Initial Committers
+Github IDs for the set of initial committers.
+- https://github.com/anandbanik
+- https://github.com/jopuneet
+- ...
```
Comment (tkuhrt) [inline labs/hlf-connector.md:21]:
```suggestion
```

File: labs/hlf-connector.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+parent: Labs
+---
+# Lab Name
+Hyperledger Fabric Connector
+
+# Short Description
+Integrate with Hyperledger Fabric using REST and Kafka and Block and Chaincode Event emitter.
+
+# Scope of Lab
+This component is design to integrate Hyperledger Fabric with enterprise systems using REST and Kafka messaging (Including Azure EventHub) with support for PDC and Transient.
```
Comment (tkuhrt) [inline labs/hlf-connector.md:13]:
```suggestion
This component is designed to integrate Hyperledger Fabric with enterprise systems using REST and Kafka messaging (Including Azure EventHub) with support for PDC and Transient.
```

**Review by tkuhrt — APPROVED** [review]
LGTM. I suggested a few minor changes, but looks like a good lab. Thanks, @anandbanik

File: labs/hlf-connector.md (hunk 1 above)
Comment (anandbanik) [reply labs/hlf-connector.md]: Removed.

**Comment by anandbanik** [comment]
Thanks @tkuhrt for you feedback and approval!

**Comment by arsulegai** [comment]
> 

Confirmed. This is a great lab proposal.

**Review by lehors — APPROVED** [review]
LGTM

**Comment by anandbanik** [comment]
Thanks @lehors  for Approval. 
@ryjones Could you please let me know the next steps. Thanks!

**Comment by ryjones** [comment]
hi @anandbanik - please transfer the repo to me, ryjones, and I will put it in the labs org

**Comment by anandbanik** [comment]
Hi @ryjones , Since this repo is an organization, i am unable to transfer it to you directly.
2 options i am aware of:-

  - Transfer the repo to hyperledger-labs directly. I believe you would have to give me access?
  - Transfer it to my personal account and then transfer it to your account.
Please let me know the option acceptable to you or any other option you are aware of.
Thanks!

**Comment by ryjones** [comment]
transfer it to your personal account, then my personal account. @anandbanik

**Comment by anandbanik** [comment]
@ryjones transferred ownership to your profile. Thanks!

**Comment by ryjones** [comment]
@anandbanik here you go! https://github.com/hyperledger-labs/hlf-connector

**Pull request merged**